### PR TITLE
Login use case for user only

### PR DESCRIPTION
**Pull request by Rebecca-Elliott**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] ✅ Test
- [ ] 🧑‍💻 Refactor

## Added/updated tests?

- [ ] Yes
- [x] No, and this is why:
- [ ] I need help with writing tests:.

## Screenshots
image

### CODE REVIEW

**Comment by ArthurAtangana** [comment]
Screenshot 2024-03-25 154023
I am supposed to manually add the user to the db before running the program?

**Review by ArthurAtangana — CHANGES_REQUESTED** [review]
Screenshot 2024-03-25 154023
it needs to loop back to login on failure i believe.

**Comment by Rebecca-Elliott** [comment]
UserDoesNotExistError has been added to help handle not finding a user in the DB.

File: sql/DML.sql
Comment (BraedenKloke) [inline sql/DML.sql:1]: @Rebecca-Elliott change 
FROM DML.sql 
TO dml.sql

File: src/main/java/Records/UserData.java
Comment (BraedenKloke) [inline src/main/java/Records/UserData.java:1]: @Rebecca-Elliott change package name
FROM Records
TO records

File: src/main/java/Utils/UserDoesNotExistError.java
Comment (BraedenKloke) [inline src/main/java/Utils/UserDoesNotExistError.java:1]: @Rebecca-Elliott change package name
FROM Utils
TO utils

**Review by BraedenKloke — APPROVED** [review]
See comments on file names to change before merging.

**Review by ArthurAtangana — APPROVED** [review]
Note to add type checks on the username for login. currently exiting on string input

**Pull request merged**